components/bms_worker/include/Sys.h

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file Sys.h
+ * @brief  Header file for System Manager
+ * @author Joshua Lafleur (josh.lafleur@outlook.com)
+ * @date 2023-12-27
+ */
+
+#pragma once
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+/**< Firmware Includes */
+#include "HW_gpio.h"
+
+/******************************************************************************
+ *                              D E F I N E S
+ ******************************************************************************/
+
+/******************************************************************************
+ *                              E X T E R N S
+ ******************************************************************************/
+
+/******************************************************************************
+ *                             T Y P E D E F S
+ ******************************************************************************/
+
+typedef uint8_t BMS_Errors_t; 
+
+typedef enum {
+    BMS_ERROR = 0x00,
+    CELL_TEMP_ERROR,
+    RH_ERROR,
+    AMBIENT_TEMP_ERROR,
+    BOARD_TEMP_ERROR,
+    ERRORS_CNT,
+} Sys_Errors_E;
```

Comment:
I've noticed this everywhere, but all the naming should be using lowerCamelCase, where snake_case should be used to separate parts of the name. For example, this should be SYS_errors. If the name were longer, you'd use camel case. Eg. SYS_errorsPlusMoreWords, etc.